apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/java/file/SchemaBuilder.kt

```diff
@@ -41,7 +42,7 @@ class SchemaBuilder(
   private fun typesFieldSpec(): FieldSpec {
     val allTypenames = interfaces.map { it.name } + objects.map { it.name } + unions.map { it.name }
     val initilizer = allTypenames.sortedBy { it }.map {
-      CodeBlock.of("$T.type", context.resolver.resolveSchemaType(it))
+      CodeBlock.of("$T.$type", context.resolver.resolveSchemaType(it))
```

Comment:
(Unrelated to this fix, but noticed we didn't use the constant here but we should)